packages/astro/src/content/content-layer.ts

```diff
@@ -180,6 +176,15 @@ export class ContentLayer {
 					}
 				}
 
+				// If loaders are specified, only sync the specified loaders
+				if (
+					options?.loaders &&
+					(typeof collection.loader !== 'object' ||
+						!options.loaders.includes(collection.loader.name))
```

Comment:
I think I understand `!options.loaders.includes(collection.loader.name)`, but I ask just in case: why do we add this check?

Reply:
Yeah, this will allow selective sync: if `loaders` is passed then only loaders named in that list will be refreshed